tools/ci/download_bin.sh

```diff
@@ -57,40 +61,73 @@ download_artifact_linux()
         echo "CircleCI build_num:${build_num} can't find artifacts."
         exit 1
     fi
-    echo -e "\033[32mDownloading binary from ${link} \033[0m"
-    cd ${output_dir} && curl -LO ${link} && tar -zxf fisco*.tar.gz && rm fisco*.tar.gz
+    LOG_INFO "Downloading binary from ${link} "
+    curl -#LO ${link} && tar -zxf fisco*.tar.gz && rm fisco*.tar.gz
```

Comment:
Codacy found an issue: [Double quote to prevent globbing and word splitting.](https://app.codacy.com/gh/FISCO-BCOS/FISCO-BCOS/pullRequest?prid=5859193)